frontend/lib/ui/dashboard/testresults_components.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:yaru_widgets/widgets.dart';
+
+import '../../models/test_results.dart';
+import '../spacing.dart';
+
+class GroupTestResultView extends StatelessWidget {
+  const GroupTestResultView({
+    super.key,
+    required this.testResultStatus,
+    required this.groupTestResults,
+  });
+
+  final List<TestResult> groupTestResults;
+  final TestResultStatus testResultStatus;
+
+  @override
+  Widget build(BuildContext context) {
+    return YaruExpandable(
+      header: Row(
+        children: [
+          testResultStatus.icon,
+          const SizedBox(width: Spacing.level4),
+          Text(
+            testResultStatus.name,
+            style: Theme.of(context).textTheme.titleLarge,
+          ),
+          const Spacer(),
+          Text(
+            groupTestResults.length.toString(),
+            style: Theme.of(context).textTheme.titleLarge,
+          ),
+        ],
+      ),
+      expandButtonPosition: YaruExpandableButtonPosition.start,
+      child: Padding(
+        padding: const EdgeInsets.only(left: Spacing.level4),
+        child: Column(
+          children: groupTestResults
+              .map(
+                (testResult) => _TestResultView(testResult: testResult),
+              )
+              .toList(),
+        ),
+      ),
+    );
+  }
+}
+
+class _TestResultView extends StatelessWidget {
+  const _TestResultView({required this.testResult});
```

Comment:
can you update the screenshot on the PR to show this widget. As I don't know what it should look like

Reply:
I have now added a few screenshots. However, since this is first time I'm using Flutter, I couldn't figure out how to align all this stuff nicely to the left, and it ends up in the centre of the dialog.